weasel/cli/push.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+from wasabi import msg
+
+import typer
+from ..util._remote_storage import RemoteStorage, get_command_hash, get_content_hash
+from ..util._general import load_project_config, Arg, logger
+
+
+def push_cli(
+    # fmt: off
+    remote: str = Arg("default", help="Name or path of remote storage"),
+    project_dir: Path = Arg(Path.cwd(), help="Location of project directory. Defaults to current working directory.", exists=True, file_okay=False),
+    # fmt: on
+):
+    """Persist outputs to a remote storage. You can alias remotes in your
+    project.yml by mapping them to storage paths. A storage can be anything that
+    the smart-open library can upload to, e.g. AWS, Google Cloud Storage,
+    local directories etc.
+
+    DOCS: https://spacy.io/api/cli#project-push
+    """
+    config = load_project_config(project_dir)
+    if "remotes" not in config:
+        # TODO: add test for this.
+        msg.fail("No `remote` configured in `project.yml` file.")
+        raise typer.Exit(code=1)
```

Comment:
```suggestion
        msg.fail("No 'remote' configured in project.yml file.", exits=1)
```

(Also in spacy the name "project.yml" is configurable, I'm not sure that's been carried over here?)

Reply:
I don't think we want to get rid of that, I'll check and make sure it's still configurable.

Reply:
Looks like that was an isolated case. There are literal references to `project.yml` throughout the code, but only in docstrings/tests, which I think is fine.